os/drivers/audio/alc5658.c

```diff
@@ -517,7 +519,7 @@ static void alc5658_set_i2s_samplerate(FAR struct alc5658_dev_s *priv)
  ****************************************************************************/
 static int alc5658_getcaps(FAR struct audio_lowerhalf_s *dev, int type, FAR struct audio_caps_s *caps)
 {
-#if !defined(CONFIG_AUDIO_EXCLUDE_VOLUME) || !defined(CONFIG_AUDIO_EXCLUDE_TONE)
+#if !defined(CONFIG_AUDIO_EXCLUDE_VOLUME) || !defined(CONFIG_AUDIO_EXCLUDE_GAIN)
```

Comment:
CONFIG_AUDIO_EXCLUDE_TON should be considered because of music player in future.

Reply:
CONFIG_AUDIO_EXCLUDE_TONE is recovered.

Reply:
you can use bracket.

Reply:
A bracket is added.